### DISCUSSION TITLE
Movie-Scrobbler: My first project in Blitzjs

### DISCUSSION THREAD

**Discussion by skorotkiewicz** [blitz-js/blitz#2485] [Show and tell]
Hello Blitzjs community!

First of all I would like to thank the author and all contributors for creating such a cool project as Blitzjs!

As soon as I saw the Blitzjs project in the suggested projects on Github a few days ago, I immediately thought I would create a "Movie Scrobbler" in it, which idea had been sitting in my head for a few weeks, but I don't start to work because I didn't want to create again user authentication, and Blitzjs solved that problem!

Therefore, I would like to show you my project: 
https://movie-scrobbler.herokuapp.com/u/skorotkiewicz

Created according to my idea, a simple timeline with watched videos (timeline like on last.fm), a list of "to watch" with the ability to click on "watched" simple and convenient with the ability to quickly rate a given video.

By the way, I also learned TypeScript, I already knew the basics, but I never wanted to get into writing in TS, and Blitzjs in a nice way forced me to write in TS.

Thanks again for creating Blitzjs!

**Comment by flybayer** [comment]
Awesome, thank you for the kind words and thank you for sharing your project!
